vars/piperPipelineStageRelease.groovy

```diff
@@ -51,15 +54,22 @@ void call(Map parameters = [:]) {
         // telemetry reporting
         utils.pushToSWA([step: STEP_NAME], config)
 
-        if (config.cloudFoundryDeploy) {
-            durationMeasure(script: script, measurementName: 'deploy_release_cf_duration') {
-                cloudFoundryDeploy script: script
+        // Prefer the newer multiCloudDeploy step if it is configured as it is more capable
```

Comment:
In this comment, it is "multiCloudDeploy", but the parameter is "multicloudDeploy" with small "c". Capital C looks better to me, but if the small "c" is intentional, the comment should be made consistent.

Reply:
I don't like the spelling of `multicloudDeploy`, but it would be a breaking change to rename it. Let's keep it consistent.